pages/entry.jsx

```diff
@@ -15,10 +15,10 @@ export default React.createClass({
   fetchData(entryId = ``) {
     Service.entry
       .get(entryId)
-      .then((entry) => {
+      .then((response) => {
         this.setState({
           loadedFromGoogle: true,
-          entry: entry
+          entry: JSON.parse(response)
```

Comment:
this needs a try/catch, otherwise this call has the potential to completely crash the app. Ideally at this point the data is already correct though.